templates/workspace_services/guacamole/guacamole-server/docker/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.3.8"
+__version__ = "0.3.9"
```

Comment:
```suggestion
__version__ = "0.4.0"
```
This is a meaningful change that should be noted by at least a minor version bump. Please read the instructions in the PR template.